lib/ar_transaction_changes.rb

```diff
@@ -1,7 +1,7 @@
 require "ar_transaction_changes/version"
 
 module ArTransactionChanges
-  if (ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR >= 2) || ActiveRecord::VERSION::MAJOR > 4
+  if Class.new { include ActiveSupport::Callbacks }.new.private_methods.include?(:_run_callbacks)
```

Comment:
`.new.private_methods.include?(:method)` can be replaced by `.private_method_defined?(:method)`